.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: java
+jdk: openjdk11
```

Comment:
I think we probably need to build the project too. You can see an example [here](https://github.com/sdkman/sdkman-broker/blob/master/.travis.yml) of what should be in a `.travis.yml` although you can probably discard the docker stuff for now.